src/Molecules/ActionCard/ActionCard.stories.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react'
+
+import ActionCard from './index'
+
+import markdown from './README.md'
+import Editable from '../Editable'
+import { FormControlLabel } from '../FormControl'
+import Checkbox from '../../Atoms/Checkbox'
+import FlexBox from '../../Templates/FlexBox'
+import Trans from '../../Atoms/Trans'
+
+import FormGroup from '@material-ui/core/FormGroup'
+import { text } from '@storybook/addon-knobs'
+import { Times } from '@styled-icons/fa-solid/Times'
+import { Check } from 'styled-icons/material/Check'
+import { Edit } from 'styled-icons/material/Edit'
+
+export default {
+  title: 'Molecules/ActionCard',
+}
+
+let originalValue
+
+const options = [
+  { value: 'option1', label: 'First option' },
+  { value: 'option2', label: 'Second option' },
+  { value: 'option3', label: 'Third option' },
+  { value: 'option4', label: 'Last option' },
+]
+
+const EditableList = ({ edit, value, onChange }) => {
+  const {
+    textOne,
+    textTwo,
+    checkbox,
+  } = value
+
+  return <FlexBox gap={3} mb={0} flexDirection="column" fontSize="body">
+    <Editable
+      edit={edit}
+      value={textOne}
+      label={text('label', 'Label')}
+      onChange={e => onChange({ ...value, textOne: e.target.value })}
+      labelSize="body"
+      descriptionSize="fontSize"
+      descriptionFontFamily="fontFamily"
+    />
+    <Editable
+      edit={edit}
+      value={textTwo}
+      label={text('label', 'Label')}
+      onChange={e => onChange({ ...value, textTwo: e.target.value })}
+      labelSize="body"
+      descriptionSize="fontSize"
+      descriptionFontFamily="fontFamily"
+    />
+    <Editable
+      edit={edit}
+      Type={FormGroup}
+      value={checkbox}
+      label={text('label', 'Label')}
+      labelSize="body"
+      descriptionSize="fontSize"
+      descriptionFontFamily="fontFamily"
+    >
+      {options.map(({ value, label }) =>
+        <FormControlLabel
+          control={
+            <Checkbox
+              value={value}
+              checked={checkbox[value]}
+              onChange={(e, checked) => onChange({ checkbox: { ...checkbox, [value]: checked } })}
+            />
+          }
+          label={label}
+          key={`checkbox ${value}`}
+        />,
+      )}
+    </Editable>
+  </FlexBox>
+}
+
+const Actions = ({
+  edit,
+  onEdit,
+  value,
+  onChange,
+}) => {
+  const onCancelClick = () => {
+    onChange(originalValue)
+    onEdit(false)
+  }
+
+  const onEditClick = () => {
+    originalValue = value
+    onEdit(true)
+  }
+
+  const onSaveClick = () => {
+    onEdit(false)
+  }
```

Comment:
We actually don't need that if  we let the story handle the state like this:
```jsx
  const [value, setValue] = useState(originalValue)

  // we pass these in Actions  
  const onSave = () => someApiCall()
  const onCancel = () => setValue(originalValue)

  <ActionCard ...></ActionCard>
```

and in Action